test/channels/retro_channel_test.exs

```diff
@@ -46,13 +46,6 @@ defmodule RemoteRetro.RetroChannelTest do
 
   describe "pushing `proceed_to_next_stage` with a stage of 'action-item-distribution'" do
     setup [:join_the_retro_channel]
-
```

Comment:
You're right in the semantic sense that this isn't a true unit test, and you're right that adding a bunch of test records would be a long walk and a bad habit to get into, but to delete the test outright is not the answer. It was small, fast, and covered a crucial side effect of this handler. Just maintain the current expectation of the `subject` field and we're golden.